jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/PacketInfo.kt

```diff
@@ -90,6 +91,16 @@ class EventTimeline(
     }
 }
 
+enum class PacketType {
+    Routed,
+    RTX,
+    Padding,
+    Synthesized,
+    RTCP,
```

Comment:
Not sure what "RTCP" means here. RTCP packets can be Routed, or Synthesized, right?

Reply:
The rate trackers I'm measuring currently are only measuring VideoType packets, so RTCP isn't really handled at all.  Possibly I should just leave RTCP as TODO for now.